ts/tsconfig.json

```diff
@@ -28,6 +28,10 @@
             "trusted-types"
         ]
     },
+    "exclude": [
+        "./Core/MSPointer.ts",
```

Comment:
@TorsteinHonsi Should we keep MSPointer in `code/es-modules`?

Reply:
Yes, I guess we still need the file, just not include it in the default build?